config.js

```diff
@@ -74,7 +74,11 @@ config.http = {
     /**
```

Comment:
An entry in CHANGES_NEXT_RELEASE regarding the changes in this PR should be added, for instance:

```
- Add: config.http.timeout (and associated enviroment variable IOTA_HTTP_TIMEOUT) (#97)
```

Reply:
Fixed in 4484f3d33ed2a680d690b8823533d9698a75c845